lib/components/viewers/trip-viewer.js

```diff
@@ -106,6 +120,17 @@ class TripViewer extends Component {
                 )}
               </RouteName>
 
+              <TripDescription>
+                <FormattedMessage
+                  id="components.TripViewer.tripDescription"
+                  values={{
+                    boardAtStop: tripData.stops[viewedTrip.fromIndex].name,
+                    disembarkAtStop: tripData.stops[viewedTrip.toIndex].name,
```

Comment:
Good idea!

You might be able to surround the contents with `<strong>` directly instead of passing a `<strong>` tag. 

Add a TODO to add links to the entry in the list of stops, so that when people navigate, they can see/hear the time they will reach the boarding and arriving stops, and also access the stop viewer links without having to go through all the stops not on the trip.